imports/plugins/custom/payments-wallet/client/checkout/components/walletCheckout.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import roundToTwo from "../../helpers/roundToTwo";
+
+class WalletCheckout extends Component {
+  state = {
+    showBody: false
+  }
+
+  handleButtonClick = () => {
+    this.props.checkout(this.props.wallet)
+      .then((walletCheckoutSuccess) => {
+        Alerts.toast(walletCheckoutSuccess.message);
+      })
+      .catch((walletCheckoutError) => {
+        Alerts.toast(walletCheckoutError.message, walletCheckoutError.type);
+      });
+  }
+
+  toggleBody = () => {
+    this.setState({ showBody: !this.state.showBody });
```

Comment:
It's preferable to use a function here since you're making use of it's previous state which might have been changed due to another event. So you could try something like `this.setState((prevState) => {........})`.